libdnf/base/goal.cpp

```diff
@@ -496,10 +496,16 @@ GoalProblem Goal::Impl::add_group_specs_to_goal(base::Transaction & transaction)
     std::vector<std::tuple<std::string, transaction::TransactionItemReason, comps::GroupQuery, GoalJobSettings>>
         groups_remove, groups_install;
     for (auto & [action, reason, spec, settings] : group_specs) {
-        bool strict = settings.resolve_strict(cfg_main);
+        bool skip_unavailable = settings.resolve_skip_unavailable(cfg_main);
         if (action != GoalAction::INSTALL && action != GoalAction::REMOVE) {
             transaction.p_impl->add_resolve_log(
```

Comment:
Because calls of add_resolve_logs() uses a different value for the last bool argument (skip_broken, skip_unavailable, ...) - I would recommend to add a description to that private method and may be not call the argument strict. What do you think?

Reply:
The only meaning of this argument is to distinguish whether the `error` or `warning` level is gonna be logged.
So I was thinking that `strict` in this case kind of make sense. How about calling it `is_error`?

Reply:
What about to use: logger::Level instead? It will be mor generic and it will state what it does.

enum class Level : unsigned int { CRITICAL, ERROR, WARNING, NOTICE, INFO, DEBUG, TRACE };

Reply:
Makes sense. Although bool option was easier to use.

Reply:
I've added a commit changing the `bool strict` argument of logging functions to `libdnf::Logger::Level log_level`.